e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -207,92 +204,17 @@
               },
               "TwinUpdateFrequency": {
                 "value": "<TwinUpdateFrequency>"
-              }
-            },
-            "settings": {
-              "image": "<Container_Registry>/microsoft/azureiotedge-twin-tester:<Build.BuildNumber>-linux-<Architecture>",
-              "createOptions": "{\"HostConfig\":{\"LogConfig\":{\"Type\":\"json-file\",\"Config\":{\"max-size\":\"4m\",\"max-file\":\"7\"}}}}"
-            }
-          },
-          "twinTester2": {
-            "version": "1.0",
-            "type": "docker",
-            "status": "running",
-            "restartPolicy": "always",
-            "env": {
-              "trackingId": {
-                "value": "<TrackingId>"
-              },
-              "testResultCoordinatorUrl": {
-                "value": "http://testResultCoordinator:5001"
               },
-              "TwinTestMode": {
-                "value": "TwinEdgeOperations"
-              },
-              "testStartDelay": {
-                "value": "<TestStartDelay>"
-              },
-              "testDuration": {
-                "value": "<TestDuration>"
-              },
-              "ServiceClientConnectionString": {
-                "value": "<IoTHubConnectionString>"
-              },
```

Comment:
Don't we need this connection string?

Reply:
Think it's actually in there it's just that the diff makes it look like it's gone. 
I went from 4 to 2 twintester modules, so the diff looks a bit funky